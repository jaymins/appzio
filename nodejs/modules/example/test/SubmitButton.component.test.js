const SubmitButton = require('../components/SubmitButton')

describe('SubmitButton', () => {
  it('Matches its working state', () => {
    const button = SubmitButton()
    expect(button).toMatchSnapshot()
  })
})
