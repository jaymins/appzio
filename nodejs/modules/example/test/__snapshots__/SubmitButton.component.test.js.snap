// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmitButton Matches its working state 1`] = `
Object {
  "content": null,
  "row_content": Array [
    Object {
      "content": "Submit",
      "id": "submit-btn",
      "onclick": Object {
        "action": "submit-form-content",
        "id": "test",
      },
      "style_content": Object {
        "background-color": "#ffcc00",
        "border-radius": 5,
        "color": "#ffffff",
        "margin": "10 20 0 20",
        "padding": "10 20 10 20",
        "text-align": "center",
        "width": "100%",
      },
      "type": "msg-plain",
    },
  ],
  "style_content": Object {
    "text-align": "center",
  },
  "type": "row",
}
`;
